compredict/client.py

```diff
@@ -192,10 +211,11 @@ def run_algorithm(self,
                       version: Optional[str] = None,
                       evaluate: bool = True,
                       encrypt: bool = False,
-                      callback_url: Optional[str] = None,
-                      callback_param: Optional[dict] = None,
+                      callback_url: Optional[Union[str, list]] = None,
+                      callback_param: Optional[Union[dict, list]] = None,
```

Comment:
same as callback_url, lets be more precise on the type of list in callback_param: `callback_param: Optional[Union[dict, List[dict]]]`

Reply:
Changed.